stdlib/public/Concurrency/Clock.cpp

```diff
@@ -41,19 +41,36 @@ void swift_get_time(
 #elif (defined(__OpenBSD__) || defined(__FreeBSD__) || defined(__wasi__))
       clock_gettime(CLOCK_MONOTONIC, &continuous);
 #elif defined(_WIN32)
-      LARGE_INTEGER freq;
-      QueryPerformanceFrequency(&freq);
-      LARGE_INTEGER count;
-      QueryPerformanceCounter(&count);
-      // Divide count (number of ticks) by frequency (number of ticks per
-      // second) to get the counter in seconds. We also need to multiply the
-      // count by 1,000,000,000 to get nanosecond resolution. By multiplying
-      // first, we maintain high precision. The resulting value is the tick
-      // count in nanoseconds. Use 128-bit math to avoid overflowing.
-      auto quadPart = static_cast<unsigned _BitInt(128)>(count.QuadPart);
-      auto ns = (quadPart * 1'000'000'000) / freq.QuadPart;
-      continuous.tv_sec = ns / 1'000'000'000;
-      continuous.tv_nsec = ns % 1'000'000'000;
+      // This needs to match what swift-corelibs-libdispatch does
+
+      // QueryInterruptTimePrecise() was added in Windows 10 and is, as
+      // the name suggests, more precise than QueryInterruptTime().
+      // Unfortunately, the symbol is not listed in any .lib file in the SDK and
+      // must be looked up dynamically at runtime even if our minimum deployment
+      // target is Windows 10.
+      typedef decltype(QueryInterruptTimePrecise) *QueryITP_FP;
+      static QueryITP_FP queryITP = nullptr;
+      static swift::once_t onceToken;
+      swift::once(onceToken, [] {
+        if (HMODULE hKernelBase = GetModuleHandleW(L"KernelBase.dll")) {
```

Comment:
Because you're directly linking mincore.lib, I'm not sure the call to `GetProcAddress()` is needed. The call is needed for `QueryUnbiasedInterruptTimePrecise()` because it's supposed to be in kernel32.lib but appears to have been accidentally omitted.

If you can just call the function directly, there's no need for the `swift::once` call or fallback to `QueryInterruptTime()`.

Reply:
I don't know if it is—we should double-check is all. :)

Reply:
Try updating the PR to just call the function directly—if it's not in the .lib file, it'll fail to build and we'll have our answer.

Reply:
I think we can call it directly; looks like both functions are in `mincore.lib` after all.